parity/cli/mod.rs

```diff
@@ -542,6 +544,7 @@ struct Mining {
 	tx_gas_limit: Option<String>,
 	tx_time_limit: Option<u64>,
 	relay_set: Option<String>,
+	min_gas_price: Option<String>,
```

Comment:
could leave `--gasprice` in legacy and introduce a new flag. our naming convention would be `--gas-price` anyway